easybuild/tools/docs.py

```diff
@@ -1199,7 +1199,7 @@ def avail_toolchain_opts_md(name, tc_dict):
     tc_items = sorted(tc_dict.items())
     table_values = [
         ['``%s``' % val[0] for val in tc_items],
-        ['%s' % val[1][1] for val in tc_items],
+        [val[1][1] for val in tc_items],
```

Comment:
no need for `str(val[1][1])`?

Reply:
`tc_dict` is dict mapping toolchain options to a tuple of `(default_value, description)`. And the description can only be string (and is for all examples I could find)

Reply:
:+1: